alembic/versions/e2c185af1226_pics_v2_support.py

```diff
@@ -0,0 +1,39 @@
+"""pics_v2_support
+
+Revision ID: e2c185af1226
+Revises: 9df8004ad9bb
+Create Date: 2024-06-19 11:46:15.158526
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "e2c185af1226"
+down_revision = "9df8004ad9bb"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "testrunexecution",
+        sa.Column("certification_mode", sa.Boolean(), nullable=False, default=False),
+    )
+    op.add_column(
+        "testsuiteexecution",
+        sa.Column("mandatory", sa.Boolean(), nullable=False, default=False),
+    )
+
+    op.execute("UPDATE testrunexecution SET certification_mode = false")
+    op.execute("UPDATE testsuiteexecution SET mandatory = false")
```

Comment:
Is this necessary? I believe that the default value is populated automatically in the existing records. Otherwise, I think we'd get an error and it would be necessary to first create the columns as nullable, update the existing records and then make it not nullable.